### PR TITLE
S10_add implement update task api

### PR DESCRIPTION
**Pull request by denys-rybachok**
Created route, controller, service for updating task.

### CODE REVIEW

File: services/task.service.js [hunk 1]
```diff
@@ -25,8 +25,29 @@ export const createTask = async (data) => {
...
   try {
     const task = await Task.findById(id);
+
     return task;
   } catch (error) {
     throw new Error(INTERNAL_SERVER_ERROR.message);
   }
 };
+
+export const updateTask = async (id, {title, priority, tags, expired_date}) => {
```
Comment (VYaniv) [inline services/task.service.js:35]: Returned type.

File: utils/handleServerError.uril.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {INTERNAL_SERVER_ERROR, NOT_FOUND_ERROR} from '../constants/errors.js';
+
+export const handleServerError = (res) => {
```
Comment (VYaniv) [inline utils/handleServerError.uril.js:3]: Returned type.

**Review by VYaniv — APPROVED** [review]

File: services/task.service.js (hunk 1 above)
Comment (denys-rybachok) [reply services/task.service.js:35]: Sorry, but I use js, not ts here. Or do you mean smth else?

Comment (VYaniv) [reply services/task.service.js:35]: All good, it is my fault.

File: utils/handleServerError.uril.js (hunk 2 above)
Comment (VYaniv) [reply utils/handleServerError.uril.js:3]: The same here.

**Pull request merged**